src/app/[lang]/profile/page.tsx

```diff
@@ -6,6 +6,8 @@ import { getServerAuthSession } from '@/lib/auth';
 import { notFound } from 'next/navigation';
 import { Locale } from '@i18n/i18n-config';
 import { useTranslation } from '@/lib/i18n';
+import { getAllUsers } from '@/lib/prisma/users';
+import CreateTag from '../admin/create-tag/page';
```

Comment:
It would be nice if we could get rid of the old create-tag-page entirely and instead use the new CreateTag-component from dashboard/CreateTag.tsx here too

Reply:
Removed the entire create-tag route as it's kind of duplicate at the moment.